### PR TITLE
hw pom

### PR DESCRIPTION
**Pull request by TaniaBystrova**

### CODE REVIEW

File: src/pageObjects/components/RegistrationPopup.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        this.emailInput = this._container.locator('#signupEmail')
+        this.passwordInput = this._container.locator('#signupPassword')
+        this.repeatPasswordInput = this._container.locator('#signupRepeatPassword')
+        this.registerButton = this._container.locator('button', {hasText: 'Register'})
+        this.errorMessage = this._container.locator('.invalid-feedback').filter({ state: 'visible' })
+    }
+
+    async getRegistrationPopup(){
+        await expect(this.registrationPopup).toBeVisible()
+    }
```
Comment (sttaran) [inline src/pageObjects/components/RegistrationPopup.js:23]: method name and method body are not matched

File: src/pageObjects/welcomePage/WelcomePage.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    constructor(page) {
+        super(page, '/', page.locator('.guest-layout'))
+        this.signUpButton = this._page.locator('button', {hasText: 'Sign Up'})
+        this.registrationPopup = this._page.locator('.modal-dialog.modal-dialog-centered')
+    }
+    
+    
+    async getRegistrationPopup(){
+        await this.signUpButton.click()
+        await expect(this.registrationPopup).toBeVisible()
```
Comment (sttaran) [inline src/pageObjects/welcomePage/WelcomePage.js:14]: remove this expect and return component RegistrationPopup

File: src/pageObjects/components/RegistrationPopup.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+//const BaseComponent = require('./BaseComponent')
+import BaseComponent from '../BaseComponent'
+const { expect } = require('@playwright/test')
+
+export default class Registration extends BaseComponent {
```
Comment (sttaran) [inline src/pageObjects/components/RegistrationPopup.js:5]: export default class RegistrationPopup

File: tests/auth/register.spec.js [hunk 4]
```diff
@@ -1,70 +1,94 @@
 const { test, expect } = require('@playwright/test');
 const { faker } = require('@faker-js/faker');
+const { default: WelcomePage } = require('../../src/pageObjects/welcomePage/WelcomePage');
+const { default: Registration } = require('../../src/pageObjects/components/RegistrationPopup');
 const userEmail = faker.internet.email()
 
-test.describe.only('Login page', () => {
-let registrationPopup
+test.describe('Login page', () => {
+let registration
```
Comment (sttaran) [inline tests/auth/register.spec.js:8]: let registrationPopup

**Review by sttaran — CHANGES_REQUESTED** [review]

File: src/pageObjects/components/RegistrationPopup.js (hunk 1 above)
Comment (TaniaBystrova) [reply src/pageObjects/components/RegistrationPopup.js:23]: fixed

File: src/pageObjects/welcomePage/WelcomePage.js (hunk 2 above)
Comment (TaniaBystrova) [reply src/pageObjects/welcomePage/WelcomePage.js:14]: fixed

**Pull request merged**